processor/src/main/kotlin/permissions/dispatcher/processor/util/Constants.kt

```diff
@@ -1,9 +1,10 @@
 package permissions.dispatcher.processor.util
 
-val FILE_COMMENT = "This file was generated by PermissionsDispatcher. Do not modify!"
-val GEN_CLASS_SUFFIX = "PermissionsDispatcher"
-val GEN_REQUEST_CODE_PREFIX = "REQUEST_"
-val GEN_PERMISSION_PREFIX = "PERMISSION_"
-val GEN_PENDING_PREFIX = "PENDING_"
-val GEN_WITH_PERMISSION_CHECK_SUFFIX = "WithPermissionCheck"
-val GEN_PERMISSION_REQUEST_SUFFIX = "PermissionRequest"
+const val DEPRECATED_MESSAGE = "android.app.Fragment became deprecated in Android P."
```

Comment:
Let's go with "**was** deprecated" instead of *became*!

Reply:
🙇